ELU.lua

```diff
@@ -6,17 +6,20 @@ local ELU, parent = torch.class('nn.ELU', 'nn.Module')
    http://arxiv.org/pdf/1511.07289.pdf
 --]]
 
-function ELU:__init(alpha)
+function ELU:__init(alpha, inplace)
    parent.__init(self)
    self.alpha = alpha or 1
    assert(type(self.alpha) == 'number')
+   self.inplace = inplace or false
+   assert(type(self.inplace) == 'boolean')
 end
 
 function ELU:updateOutput(input)
    input.THNN.ELU_updateOutput(
       input:cdata(),
       self.output:cdata(),
-      self.alpha
+      self.alpha,
+      self.inplace
```

Comment:
prob change this to: self.inplace or false (as saved models wont have the .inplace member)